src/test/java/com/hocs/test/pages/markup/MarkUpDecision.java

```diff
@@ -152,28 +109,16 @@ public void assertTopicIsRequiredErrorMessage() {
         assertThat(topicIsRequiredErrorMessage.getText(), is("Topic is required"));
     }
 
-    public void getToMarkupAddATopicScreenPrerequisites() {
-        clickOn(policyResponseRadioButton);
-        clickOn(continueButton);
-        sleep(500);
+    public void assertNRNTextBoxIsDisplayed() {
+        Assert.assertThat(isElementDisplayed(noResponseNeededTextField), is(true));
```

Comment:
`Assert` part shouldn't be needed here. Code should still work starting from `assertThat`